.gitmodules

```diff
@@ -1,23 +1,23 @@
 [submodule "MCServer/Plugins/Core"]
-	path = MCServer/Plugins/Core
+	path = Server/Plugins/Core
 	url = https://github.com/mc-server/Core.git
 [submodule "MCServer/Plugins/ProtectionAreas"]
```

Comment:
Never. Once the submodules have been named like this, they need to stay or else you break everyone's checked-out repo.

Since this is only a git-internal name for the submodule and not the actual path, it's of no consequence, really. On the disk the files will be checked out to the proper path.